tests/framework/extensions/rancherversion/rancherversion.go

```diff
@@ -0,0 +1,19 @@
+package rancherversion
+
+import (
+	"io/ioutil"
+	"net/http"
+)
+
```

Comment:
Please put a comment here describing the function, so we can have a google doc on it.